src/common/KernelBase.hpp

```diff
@@ -285,6 +354,13 @@ class KernelBase
   std::vector<RAJA::Timer::ElapsedType> min_time[NumVariants];
   std::vector<RAJA::Timer::ElapsedType> max_time[NumVariants];
   std::vector<RAJA::Timer::ElapsedType> tot_time[NumVariants];
+
+#ifdef RAJAPERF_USE_CALIPER
+  bool doCaliperTiming = true; // warmup can use this to exclude timing
+// we need a Caliper Manager object per variant
+// we can inline this with c++17
+  static std::map<rajaperf::VariantID, cali::ConfigManager> mgr;
```

Comment:
Should this be an array of size NumVariants? Do we only want to create a Caliper Manager object if we're actually running a test?

Reply:
@MrBurmark for your first question are you thinking it would work better to modify the init of the managers, or is it more map vs vector syntax.  On the second can you expand on your thoughts regarding manager object when not running a test.

Reply:
@jonesholger the idea here is that there will be a cali::ConfigManager object for each variant in the Suite, correct? I think what @MrBurmark was asking about is whether it would be better to have an array of cali::ConfigManager pointers(?) of length NumVariants and only create and hold a Manager object in the array slots for the variants that are actually run during the execution of the Suite.

Reply:
Yep, a ConfigManager object per variant. The map currently only holds variants that are able and set to run. Haha, now you got me second guessing myself wrt  -ev. I'll take a look.

Reply:
mgr.size() reflects only the variants that are run. Confirmed with -ev